packages/block-library/src/list-item/edit.js

```diff
@@ -71,7 +72,7 @@ export default function ListItemEdit( {
 	const onMerge = useMerge( clientId );
 	return (
 		<>
-			<li { ...innerBlocksProps }>
+			<li { ...innerBlocksProps } ref={ useCopy( clientId ) }>
```

Comment:
Are we sure that `ref` isn't (and won't ever) already be set by `innerBlocksProps`?